### ISSUE TITLE
There any way to remove the permanent ban!?

### ISSUE THREAD

**Issue by Juan44** [finndev/PokeBuddy#1139]
( Failed to get game data from the server )

Thanks.

**Comment by protected22** [comment]
got same problem.   an hour earlier everything works. but now it looks like i got banned

(i was in same location als my bot
my bot was turned off)

**Comment by MariaBitterlich** [comment]
There's an appeal site for accounts banned by mistake - you can still try...

**Comment by protected22** [comment]
oh, where can i find it?

**Comment by TusnamiKana** [comment]
here：https://support.pokemongo.nianticlabs.com/hc/en-us/requests/new?ticket_form_id=233187
I've got unban after summit the form @@ Maybe it works on you too

**Comment by Rab89** [comment]
tbh the OP was a legit botter hes been banned simply for botting, do you really think they are gonna revert their decision for a specified high priority target lol

**Comment by Deezee11** [comment]
@TusnamiKana TusnamiKana

How long did it take you to get unbanned? Did they write back to you or just unban you without any notifications?

Thanks I want to know because I got a ban now for over 24 hours.

Much appreciated!

**Comment by TusnamiKana** [comment]
@Deezee11 It take me about 2 days.They send me a email.Hope that helps you :)

**Comment by Deezee11** [comment]
@TusnamiKana 

Thank you! 

Do you mind sharing the email you received you can sensor the necessary areas you wish. Much appreciated once again!

**Comment by kenwang911** [comment]
@TusnamiKana 
Thanks, I got the same problem, if you don't mind share the email, we can see what they said....please:)

**Comment by Artillery101** [comment]
Trainer,

We have received your appeal. Our goal is to provide a fair, fun and legitimate game experience for everyone. Use of any other third-party software or app, including but not limited to scripts/apps, applications used to falsify your location, using emulators, modified or unofficial software and/or accessing Pokémon GO clients or backends in an unauthorized manner is considered cheating.

Please consider your account permanently terminated unless we reply back to your appeal.

-NianticOps

**Comment by OsOCooL** [comment]
^ i got that email LOL

**Comment by TheNapper** [comment]
Looks like I have the full ban - Won't even let me make a new account. Do you think they ban the MAC of the phone being used? 

![11](https://cloud.githubusercontent.com/assets/21034683/17659677/19fc6a0e-62cc-11e6-9f3e-66e9e53a60cf.JPG)

**Comment by OsOCooL** [comment]
What did you say on your appeal @TusnamiKana

**Comment by OsOCooL** [comment]
@TheNapper You should be able to make a new account. I made another one with the PTC and one with another gmail.

**Comment by Juan44** [comment]
I already submited my appeal wish me luck :)

**Comment by TheNapper** [comment]
I was genuine level 24 and boosted to level 30 over two weeks. Never moved location other than my local town.

**Comment by Juan44** [comment]
@TusnamiKana what did says the mail niantic sent you???

**Comment by OsOCooL** [comment]
@TheNapper Submit a appeal and GL

**Comment by TusnamiKana** [comment]
Sorry everyone I've already deleted my mail. Can't help you guys @@

**Comment by OsOCooL** [comment]
@TusnamiKana do you semi remember what you said? lol

**Comment by k1ndskopf** [comment]
@Juan44 may i ask you what you sent them? :D
Sorry for botting, please unban me?^^

**Comment by bhembenk** [comment]
@TusnamiKana 
yeah, maybe lots of us that already banned can use your words too.

**Comment by Deezee11** [comment]
@TusnamiKana  Yup that will be of help.

A short summary?
